Jenkinsfile_nightly

```diff
@@ -8,10 +8,15 @@ properties([
 @Library("Infrastructure")
 
 def type = "java"
-def product = "rpe"
+def product = "hmc"
 def component = "spring-boot-template"
 
-withNightlyPipeline(type, product, component) {}
+withNightlyPipeline(type, product, component) {
+  enableFortifyScan('hmc-aat')
```

Comment:
You don't need to pass parameter, if it is the same with product name.

Reply:
I am also seeing below error. This vault needs to be defined in Azure.

org.jenkinsci.plugins.azurekeyvaultplugin.AzureKeyVaultException: Failed to retrieve secret fortify-on-demand-username from vault https://hmc-aat.vault.azure.net, error message: Status code 404, "{"error":{"code":"SecretNotFound","message":"A secret with (name/id) fortify-on-demand-username was not found in this key vault. If you recently deleted this secret you may be able to recover it using the correct recovery command. For help resolving this issue, please see https://go.microsoft.com/fwlink/?linkid=2125182"}}"